packages/service-survey/src/controllers/statistics.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    Answer,
+    AnswerEnps,
+    AnswerType,
+    enps,
+    Surveys
+} from "@hive/lib-survey";
+import { Request, Response } from "express";
+import { getMongoCollection } from "../mongodb";
+import { HTTPResponse } from "../response";
+
+const isEnpsAnswer = (answer: Answer): answer is AnswerEnps =>
+    answer.type === AnswerType.enum.Enps;
+
+/**
+ * Calculates the eNPS score for a given survey
+ *
+ * @param _req Express response
+ * @param res Express request
+ */
+export const calculateEnps = async (
+    { params: { surveyId } }: Request<{ surveyId: string }>,
+    res: Response
+): Promise<void> => {
+    const allSurveys = Surveys.parse(
```

Comment:
We should fetch single survey by id from the database here. i.e 
`Survey.findOne({_id : surveyId })`